auto/auto.cjs

```diff
@@ -1,6 +1,6 @@
-const exports = require('../dist/chart.cjs');
-const {Chart, registerables} = exports;
+const chartjs = require('../dist/chart.cjs');
+const {Chart, registerables} = chartjs;
 
 Chart.register(...registerables);
 
-module.exports = Object.assign(Chart, exports);
+module.exports = Object.assign(Chart, chartjs);
```

Comment:
Isn't the 'dist/chart.cjs' generated from https://github.com/chartjs/Chart.js/blob/master/src/index.umd.ts? and should thus be directly compatible.

If that's the case, then

```suggestion
const Chart = require('../dist/chart.cjs');
module.exports = Chart;
```

Reply:
@kurkle `chart.cjs` generates from `index.ts`

Reply:
Is there a reason for that?

Reply:
@kurkle UMD bundle registers all components. The CommonJS bundle should work the same as the ESM bundle.

Reply:
Ok, but I still question the need for the Object.assign (from freshly built dist/chart.cjs):
![image](https://user-images.githubusercontent.com/27971921/208071807-84963768-b921-4246-ba99-be132950fb32.png)

Reply:
So I think this should also work, no?:
```js
const Chart = require('../disct/chart.cjs');
Chart.register(...Chart.registerables);
module.exprts = Chart;
```

Reply:
@kurkle it works for ESM:
```
import Chart from 'chart.js/auto'
import {Filler} from 'chart.js/auto'
```
but without Object.assign this won't work:
```
const Chart = require('chart.js/auto') // will work
const {Filler} = require('chart.js/auto') // will not
```

Reply:
@kurkle sorry

```
const Chart = require('../disct/chart.cjs'); // <-- here Chart is not Chart class
Chart.register(...Chart.registerables); // so here there no register method
```

Reply:
Thanks, got it :)